### ISSUE TITLE
No module named 'ConfScorer'

### ISSUE THREAD

**Issue by Neo-Flash** [benoitba/pytorch_dl4chem-geometry#1]
hi!
thank u for sharing ur work
but there are some files u didn't provide
I can't run your notebook after clone ur repo
could u please provide the full project?

**Comment by bbaillif** [comment]
Hi !
Sorry for the late reply, I added the ConfScorer file.
I forgot that I used it in this project, the ConfScorer object was made to analyze one or multiple molecules with different sets of conformations, to compare between conformations generated/obtained in different ways. I planned to make it a standalone package.
On a more general note, this implementation was shown to give rather flat molecules (as shown in the PMI plots in the generated_test_molecules_COD.ipynb), but I can't be sure whether I failed in my reimplementation, or if it's an actual caveat of the DL architecture that makes the model optimize the loss by reproducing correctly 2 dimensions (XY) and not all 3 (XYZ) in the RMSD calculation. I would therefore suggest to switch to new implementations such as GeoMol or more recent (https://arxiv.org/abs/2202.01356, which benchmarks against this model that has poor performances) to generate conformations with DL.
Cheers,
Benoit
